changelog/fix_make_redundant_percent_q_safe_on_multiline_strings.md

```diff
@@ -0,0 +1 @@
+* [#12789](https://github.com/rubocop/rubocop/pull/12789): Make `Style/RedundantPercentQ` safe on multiline strings. ([@boardfish][])
```

Comment:
```suggestion
* [#12789](https://github.com/rubocop/rubocop/pull/12789): Fix a false positive for `Style/RedundantPercentQ` when multiline strings containing hash rocket syntax. ([@boardfish][])
```

Reply:
Hash rocket syntax isn't specifically the problem here – the issue is with multiline strings in general. I experienced it on a string with hash rocket syntax, but it actually affects any multiline use of %Q.

```diff
-%Q(
+"
  "Something is wrong"
-)
+"
```